test/common/upstream/upstream_impl_test.cc

```diff
@@ -1537,22 +1540,21 @@ TEST_F(HostImplTest, CreateConnectionHappyEyeballs) {
   locality.set_sub_zone("world");
   Network::Address::InstanceConstSharedPtr address =
       Network::Utility::resolveUrl("tcp://10.0.0.1:1234");
+  AddressVector address_list = {
+      address,
+      Network::Utility::resolveUrl("tcp://10.0.0.1:1235"),
+  };
   auto host = std::make_shared<HostImpl>(
       cluster.info_, "lyft.com", address,
       std::make_shared<const envoy::config::core::v3::Metadata>(metadata), 1, locality,
       envoy::config::endpoint::v3::Endpoint::HealthCheckConfig::default_instance(), 1,
-      envoy::config::core::v3::UNKNOWN, simTime());
+      envoy::config::core::v3::UNKNOWN, simTime(), address_list);
```

Comment:
If we don't have address_list in the HostDescriiptionImpl, then we don't have to pass in it here.